pkgs/brumhard/krewfile/pkg.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - name: github.com/brumhard/krewfile@v0.3.0
```

Comment:
```suggestion
  - name: brumhard/krewfile@v0.3.0
```

Reply:
Rename the package to `brumhard/krewfile` and set `repo_owner` and `repo_name`.
This change enables us to update the package by Renovate and enables aqua to get the package versions from GitHub Releases.